src/components/Footer/Footer.tsx

```diff
@@ -22,17 +22,18 @@ type FooterProps = LangSwitchProps & {
   postLoginLinks: Array<FooterLinksType>;
   preLoginLinks: PreLoginFooterLinksType;
   legalInfo: JSX.Element | Array<JSX.Element>;
-  onExit?: (href: string, linkType: LinkType) => void;
+  onExit?: (linkType: LinkType, href?: string) => void;
```

Comment:
it's not recommended to change the signature of a method inside a Prop, you risk to break all the application that are using it!